packages/css/src/components/shopping-cart/shopping-cart.template.ts

```diff
@@ -0,0 +1,139 @@
+import { ShoppingCart } from '@dso-toolkit/sources';
+
+import { html, nothing } from 'lit-html';
+
+import { buttonTemplate } from '../button/button.template';
+import { iconTemplate } from '../icon/icon.template';
+
+function accumulateItems(items: any) {
+  return items.reduce((t: any, item: any) => t.concat(item.subitems), []);
+}
+
+function countSubitems(items: any) {
+  return accumulateItems(items).length;
+}
+
+function hasWarning(items: any) {
+  return items && (items.some((a: any) => a.subitems) ? accumulateItems(items) : items)
+    .some((a: any) => a.warning);
+}
+
+export function shoppingCartTemplate({ collapsed, hideSummary, isOpen, items, shoppingcartTitleTag, shoppingcartTitle }: ShoppingCart) {
+  return html`
+    ${ shoppingcartTitleTag == 'h2'
+      ? html`
+        <h2>${shoppingcartTitle}</h2>
+      `
+      : html`
+        <h3>${shoppingcartTitle}</h3>
+      `
+    }
+
+    <div class="dso-shopping-cart">
+      <div class="dso-contents">
+        ${items.length > 0
+          ? html`
+            ${!hideSummary
+              ? html`
+                <button type="button" class="dso-status">
+                  ${!collapsed ? html`
+                    ${iconTemplate({ icon: 'chevron-up' })}
+                  `
+                  : html`
+                    ${iconTemplate({ icon: 'chevron-down' })}
+                  `}
+                  U heeft ${countSubitems(items)} activiteit${((countSubitems(items) > 1) ? html`en` : nothing )} gekozen
+                </button>
+              `
+              : nothing
+            }
+            ${!collapsed
+              ? html`
+                <ul class="dso-items">
+                  ${items.map(item => html`
+                    <li>
+                      ${(hasWarning(item.subitems))
+                        ? html`
+                          ${iconTemplate({ icon: 'status-warning' })}
+                          <span class="sr-only">waarschuwing</span>
+                        `
+                        : nothing
+                      }
+                      ${item.label}
+                      ${!item.readonly && !item.edit && item.additive
+                        ? html`
+                          <span class="dso-additive">${item.additive}</span>
+                        `
+                        : nothing
+                      }
+                      ${!item.readonly
+                        ? html`
+                          ${item.edit
+                            ? html`
+                              <div class="dso-edit-additive">
+                                <label for="additive-${item.id}">Bewerk toevoeging</label>
+                                <input type="text" class="dso-edit-name" id="additive-${item.id}" value="${item.additive}" />
```

Comment:
```suggestion
                                <input type="text" class="dso-edit-name" id="additive-${item.id}" value=${item.additive} />
```

Graag ook de rest van de files checken: Als je direct een interpolated value in een attribute stopt zijn de quotes niet nodig (en worden zelfs afgeraden door het lit team)